### PR TITLE
Fixed python3 issue, and windows compat issue

### PR DESCRIPTION
**Pull request by vvalien**
Print should always be print(data) in 2 and 3 for compatibility reasons.

Commented out os.geteuid as its not a windows module.

### CODE REVIEW

**Comment by zerosum0x0** [comment]
That print is just a debug statement for trying to fix #1. I would merge that commit if it was separate.

But the euid just tells you if the program is run as root. There is also readline library used throughout which I don't think works on Windows either. Or does Koadic run on windows without the euid?

**Comment by vvalien** [comment]
Yep it runs on windows, I thought this was actually intentional.

You will need to have pyreadline which is the windows port of readline, however its generally good form to install this regardless (if your on windows). Granted I haven't checked it in depth but it does run.

Also maybe send me a dm on twitter, I'd like to know how your dev'ing the jscript stuff on windows. Might save me a lot of time lol.

File: core/colors.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def colorize(self, text, options, readline=False):
 
     def get_prompt(self, state, isreadline = True):
         import os
-        glyph = "#" if os.geteuid() == 0 else "$"
+        glyph = "#" # if os.geteuid() == 0 else "$"
```
Comment (zerosum0x0) [inline core/colors.py]: We should keep the behavior on non-Windows systems

```
import platform
if platform.system() != "Windows"
```

Or just a try block... maybe that's cleaner.

**Comment by vvalien** [comment]
Sorry im like a total noob when it comes to using git

**Pull request closed without merging**